requirements.txt

```diff
@@ -16,6 +16,13 @@ requests==2.27.1
 redis==4.3.4
 fakeredis==1.9.0
 
+# Mypy and dependencies
```

Comment:
Should these be in requirements.dev.txt instead?  They are requirements when doing development, not for when running the app on GAE.

Reply:
Yes, you're right! I made this mistake in a separate `package.json` file recently as well 😓